src/LongBaseSequence.hpp

```diff
@@ -43,15 +43,22 @@ class shasta::LongBaseSequenceView {
 public:
     uint64_t* begin;
     uint64_t baseCount;
+    bool readOnly;
 
     LongBaseSequenceView(
         uint64_t* begin,
         uint64_t baseCount) :
-        begin(begin), baseCount(baseCount)
+        begin(begin), baseCount(baseCount), readOnly(false)
+    {}
+
+    LongBaseSequenceView(
+        const uint64_t* begin,
+        const uint64_t baseCount) :
+        begin(const_cast<uint64_t*>(begin)), baseCount(baseCount), readOnly(true)
     {}
 
     LongBaseSequenceView() :
-        begin(0), baseCount(0)
+        begin(0), baseCount(0), readOnly(false)
```

Comment:
Perhaps set `readOnly(true)` here, to make it more likely that an assertion occurs if attempting to use a default-constructed `LongBaseSequenceView`.

Reply:
Good point. Will try it out.